app/code/extension/SiteConfigExtension.php

```diff
@@ -128,5 +131,16 @@ public function updateCMSFields(FieldList $fields)
         $sponsorLogo2->getValidator()->setAllowedExtensions(['jpg','jpeg','png']);
         $sponsorLogo3->getValidator()->setAllowedExtensions(['jpg','jpeg','png']);
         $sponsorLogo4->getValidator()->setAllowedExtensions(['jpg','jpeg','png']);
+
+        // Alert Tab
+        $fields->addFieldToTab('Root.Alert', TextField::create(
```

Comment:
Can be refactored using `addFieldsToTab` instead of having multiple calls to the same banner. Refer to my `homepage.php` to see how I did it.

Reply:
Will do 👍